nrf_rpc/CMakeLists.txt

```
@@ -0,0 +1,16 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+zephyr_include_directories(include)
+
+zephyr_library()
+zephyr_library_sources(nrf_rpc.c nrf_rpc_cbor.c)
+
+if (CONFIG_NRF_RPC_CBOR)
+	zephyr_library_sources(nrf_rpc_cbor.c)
+endif()
```

You can use the short form:
```suggestion
zephyr_library_sources_ifdef(CONFIG_NRF_RPC_CBOR nrf_rpc_cbor.c)
```